exercises/concept/weighing-machine/WeighingMachineTests.cs

```diff
@@ -5,76 +5,49 @@
 public class WeighingMachineTests
 {
     [Fact]
-    public void Set_weight_and_get_weight()
-    {
-        var wm = new WeighingMachine();
-        wm.InputWeight = 60m;
-        Assert.Equal(60m, wm.InputWeight, precision: 3);
-    }
-
-    [Fact(Skip = "Remove this Skip property to run this test")]
-    public void Negative_weight_is_invalid()
+    public void Get_Precision()
     {
-        var wm = new WeighingMachine();
-        Assert.Throws<ArgumentOutOfRangeException>(() => wm.InputWeight = -10);
+        var wm = new WeighingMachine(precision:2);
+        Assert.Equal(2, wm.Precision);
     }
 
     [Fact(Skip = "Remove this Skip property to run this test")]
-    public void Default_Unit_Is_Kilogram()
-    {
-        var wm = new WeighingMachine();
-        
-        Assert.Equal(Unit.Kilograms, wm.Unit);
-    }
-
-    [Fact(Skip = "Remove this Skip property to run this test")]
-    public void Get_us_display_weight_pounds()
-    {
-        var wm = new WeighingMachine();
-        wm.InputWeight = 60m;
-        Assert.Equal(132, wm.USDisplayWeight.Pounds);
-    }
-
-    [Fact(Skip = "Remove this Skip property to run this test")]
-    public void Get_us_display_weight_ounces()
+    public void Set_weight_and_get_weight()
     {
         var wm = new WeighingMachine();
-        wm.InputWeight = 60m;
-        Assert.Equal(4, wm.USDisplayWeight.Ounces);
+        wm.Weight = 60m;
```

Comment:
As mentioned before, all the `m` suffixes should be removed as we're using integers now and not `decimals.`

Reply:
The precision would not make sense with ints.